src/spellbot/enums.py

```diff
@@ -114,4 +115,5 @@ def __str__(self) -> str:
     GameFormat.VINTAGE,
     GameFormat.SEALED,
     GameFormat.BRAWL_TWO_PLAYER,
+    GameFormat.PAUPER_EDH,
```

Comment:
What do you think about moving this to after CEDH? That way it's closer to all the other EDH formats.

Reply:
Good point. :)